browser/ui/BUILD.gn

```diff
@@ -180,6 +180,15 @@ source_set("ui") {
       "views/web_discovery_dialog_view.h",
     ]
 
+    # Enable first_run dialog on Win.
+    # Upstream only includes it on mac/Linux.
+    if (is_win) {
+      sources += [
+        "//chrome/browser/ui/views/first_run_dialog.cc",
+        "//chrome/browser/uiviews/first_run_dialog.h",
```

Comment:
`ui/views`

Reply:
These two files are upstream file.
So, I think it path should be absolute?

Reply:
yep, it should be absolute. there's a typo: `uiviews` instead of `ui/views` I think.

Reply:
Oops! :)

Reply:
Interesting. this typo didn't give gn error.